src/components/navigation/_navigation.scss

```diff
@@ -141,4 +141,23 @@
     &__item--active > &__link {
         font-weight: $font-weight-bold;
     }
+
+    &-wrapper {
+        &--neutral {
+            background: var(--ons-color-grey-75);
+            .ons-navigation__item {
+                &--active,
+                &:hover {
+                    border-color: var(--ons-color-white);
+                }
+                .ons-navigation__link {
+                    color: var(--ons-color-white);
+                    &:focus {
+                        background-color: var(--ons-color-focus);
+                        color: var(--ons-color-text-link-focus);
+                    }
+                }
+            }
+        }
+    }
```

Comment:
This section is a duplicate of one at the top. Any extra css needed should be added to that one rather than creating a new section